core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1035,6 +1035,18 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_RPC_READ_COUNT =
+      new Builder("Worker.RpcReadCount")
+          .setDescription("The number of active read-RPCs managed by this worker")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(true)
+          .build();
+  public static final MetricKey WORKER_RPC_WRITE_COUNT =
+      new Builder("Worker.RpcWriteCount")
+          .setDescription("The number of active write-RPCs managed by this worker")
+          .setMetricType(MetricType.COUNTER)
+          .setIsClusterAggregated(true)
```

Comment:
@ggezer From what I understand being cluster aggregated would allow you to see the total number of active worker RPCs (reads/writes) from a single metrics sink (i.e: a master node). I think that'd be worthwhile to include?

I'm not 100% clear on the implementation, but I believe the metric on the master would get updated each time any individual worker's counter gets updated (so asynchronously). @yuzhu do you know?

Reply:
I think there is some value in this, because a very high worker read count would indicate workers are overloaded and may serve as an indicator to the users to increase the number of workers. 

aggregated metrics are updated with the heartbeat messages i think.

Reply:
@ggezer Can you re-review this PR?

Reply:
Main concern I have was to avoid providing redundant metrics, as this particular one will be categorized further with load tracking metrics. However, it's OK to get this in if we're not having those soon.

Reply:
From discussion with @ggezer we'll be forgoing adding load tracking metrics for the time being; so I think these metrics do not overlap with the metrics we discussed about adding Counters to `BlockReader` implementations right?

Reply:
Here is the PR for the changes mentioned in my previous comment: https://github.com/Alluxio/alluxio/pull/13502